coops-ui/src/resources/config-objects.ts

```diff
@@ -0,0 +1,68 @@
+
+export class Business {
+     typeEnum: string;
+     displayName:String;
+     flows: Array<Flow>;
+}
+
+export class Flow {
+     feeCode:String;
+     displayName:String;
+     certifyText:String;
+     // fields:Array<Fields>
+}
+
+export const configJson = [{
+  typeEnum: 'BC',
+  displayName: 'Benefits Company',
+  flows: [
+    {
+      feeCode: 'OTADD',
```

Comment:
We just recently introduced an enum for all the filing codes i believe, you could definitely use that for all the codes!